Solutions/LargestPrimeFactor.cs

```diff
@@ -0,0 +1,36 @@
+namespace Solutions;
+
+public class LargestPrimeFactor : ProblemSolverBase
+{
+  public LargestPrimeFactor()
+  {
+    ProblemID = 3;
+  }
+  public override void Solve()
+  {
+    Answer = GetLargestPrimeFactor(600851475143).ToString();
+  }
+
+  long GetLargestPrimeFactor(long number)
```

Comment:
Let's shoot for a generic approach instead of solving individual issues here so you can learn more about how to structure a project.

Can you move this into a utility folder or file and import it here? 

Anything that can be re-used should go into a relevant utility file and these "problem" file that holds the answers to individual question should have as little generic code as they can and they should only include the "domain specific" code.